### PR TITLE
Chore/refacto organisation section

### PR DESCRIPTION
**Pull request by Isalafont**

### CODE REVIEW

**Comment by Isalafont** [comment]
⚠️ Rebase first  #572 then merge.

File: frontend/src/components/organisms/form-sections/OrganisationSection/PersonalInformationCard.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      // note that they might be more than one demandeur
+      // for now we just display the first demandeur found
+      setPersonalInformation(firstDemandeur);
+    }
+  }, [team_members]);
+
+  const onUpdatePersonalInformation = () =>
+    setUrlForDisconnectionPrompt(`${BACK_HOST}/api/users/personal_information`);
+
+  // const onDisconnectionModal = () =>
```
Comment (rdubigny) [inline frontend/src/components/organisms/form-sections/OrganisationSection/PersonalInformationCard.js:39]: remove comment

File: frontend/src/components/organisms/form-sections/OrganisationSection/DisconnectionModalCard.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import ConfirmationModal from '../../ConfirmationModal';
+
+export const DisconnectionModalCard = ({
+  urlForDisconnectionPrompt,
+  setUrlForDisconnectionPrompt,
```
Comment (rdubigny) [inline frontend/src/components/organisms/form-sections/OrganisationSection/DisconnectionModalCard.js:5]: the setUrlForDisconnectionPrompt props should not be necessary. Plz remove this and replace it with more readable props. (PP session might be usefull here)

**Review by rdubigny — CHANGES_REQUESTED** [review]

File: frontend/src/components/organisms/form-sections/OrganisationSection/OrganisationCard.js [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+import Loader from '../../../atoms/Loader';
+import { Card, CardHead } from '../../../molecules/Card';
+import { FormContext } from '../../../templates/Form';
+import { useAuth } from '../../AuthContext';
+import DisconnectionModalCard from './DisconnectionModalCard';
+import OrganizationPrompt from './OrganizationPrompt';
+
+const { REACT_APP_BACK_HOST: BACK_HOST } = process.env;
+
+export const OrganisationCard = ({}) => {
```
Comment (rdubigny) [inline frontend/src/components/organisms/form-sections/OrganisationSection/OrganisationCard.js:16]: remove empty object

**Review by rdubigny — CHANGES_REQUESTED** [review]

**Review by rdubigny — APPROVED** [review]

**Pull request merged**